frontend/src/libs/helpers/generate-uuid.helper.ts

```diff
@@ -0,0 +1,5 @@
+const generateUUID = (): string => {
+  return crypto.randomUUID();
```

Comment:
crypto is node lib, not sure that is available in frontend

Reply:
<img width="845" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/bc83e18d-c79c-4934-ab39-45105183bd9f">
 in 94.77%